Common/DTO/UserCredential.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DTO
+{
+    /// <summary>
+    /// Minimal model of user: contains email and password
+    /// </summary>
+    public class UserCredential
+    {
+        public UserCredential(string email, string password)
+        {
+            Email = email;
+            Password = password;
+        }
+        public string Email { get; set; }
+        public string Password { get; }
```

Comment:
I don't understand why password here and password in User model?

Reply:
Why is it bad?

Reply:
Edit function can edit password and other data. But we have to transmit the same object - user. So I think that logic would be the same if we edit password and name, for example.